cmd/ops/deprecation.go

```diff
@@ -97,3 +110,12 @@ func deprecationJsonOut(queryFields parser.QueryFieldList) error {
 	fmt.Print(string(bytes))
 	return nil
 }
+
+func deprecationXcodeOut(queryFields parser.QueryFieldList) {
```

Comment:
For now, this is just for my script in xcode, but I think we should promote this to be `plaintext` output, which will be simpler, something like `file:line | field | reason`